tests/ui/test_toc.py

```diff
@@ -0,0 +1,35 @@
+from tests.ui.pages.content import Content
+from tests.ui import markers
+
+
+@markers.parametrize("book_slug,page_slug", [("college-physics", "preface")])
+@markers.nondestructive
+def test_toc_toggle_button_opens_and_closes(selenium, base_url, book_slug, page_slug):
```

Comment:
@TomWoodward Here I added a basic test of clicking the TOC toggle button. This test currently passes for both desktop and mobile resolutions. However, in the mobile view it really covers up the screen so I would suggest that the mobile table of contents be hidden by default.

Reply:
i believe this is the plan you can see the upcoming new designs here openstax/unified-issues#104